pages/dao/[symbol]/proposal/components/instructions/splTokenTransferForm.tsx

```diff
@@ -0,0 +1,229 @@
+                validateTokenAccountMint(
+                  destAccMint,
+                  sourceAccMint?.account.mint
+                )
+                return true
+              } else {
+                return false
+              }
+            }
+          } catch (e) {
+            return false
+          }
+          return false
+        }
+      ),
+    amount: yup.string().required('Amount is required'),
+    sourceAccount: yup
+      .object()
+      .nullable()
+      .required('Source account is required'),
+  })
+  const handleSetForm = ({ propertyName, value }) => {
+    setFormErrors({})
+    setForm({ ...form, [propertyName]: value })
+  }
+  const setMintInfo = (value) => {
+    setForm({ ...form, mintInfo: value })
+  }
+  const setamount = (event) => {
+    const { min, max } = event.target
+    let value = event.target.value
+    value = !value
+      ? ''
+      : Math.max(Number(min), Math.min(Number(max), Number(value))).toFixed(
+          currentPrecision
+        )
+    handleSetForm({
+      value: value,
+      propertyName: 'amount',
+    })
+  }
+  const validateInstruction = async () => {
+    const { isValid, validationErrors } = await isFormValid(schema, form)
+    setFormErrors(validationErrors)
+    return isValid
+  }
+  async function getSerializedInstruction() {
+    const isValid = await validateInstruction()
+    let serializedInstruction = ''
+    if (isValid && programId && form.sourceAccount?.pubkey) {
+      const mintAmount = parseMintNaturalAmountFromDecimal(
+        form.amount,
+        mintMinAmount
+      )
+      const transferIx = Token.createTransferInstruction(
+        programId,
+        form.sourceAccount?.pubkey,
+        new PublicKey(form.destinationAccount),
+        form.sourceAccount?.info?.governedAccount,
+        [],
+        mintAmount
+      )
+      serializedInstruction = serializeInstructionToBase64(transferIx)
+    }
+    const obj: Instruction = {
+      serializedInstruction,
+      isValid,
+      sourceAccount: form.sourceAccount,
+    }
+    return obj
+  }
+
+  useEffect(() => {
+    handleSetForm({
+      propertyName: 'programId',
+      value: programId?.toString(),
+    })
+  }, [realmInfo?.programId])
+  useEffect(() => {
+    async function tryGetTooknAccount() {
+      if (form.sourceAccount?.info.governedAccount) {
+        const mintResponse = await tryGetTokenMint(
+          connection.current,
+          form.sourceAccount?.info.governedAccount
+        )
+        setMintInfo(mintResponse?.account)
+      }
+    }
+    if (form.sourceAccount?.info.governedAccount) {
+      tryGetTooknAccount()
+    }
+  }, [form.sourceAccount?.pubkey])
+  useImperativeHandle(ref, () => ({
+    getSerializedInstruction,
+  }))
+
+  return (
+    <div className="mt-5">
+      <div>Program id</div>
+      <div>{form.programId}</div>
+      <div>Account owner (governance account)</div>
+      <div>{form.sourceAccount?.pubkey?.toString()}</div>
```

Comment:
There is no need to display `Program id` and` Account owner (governance account)`